example/main.tf

```diff
@@ -0,0 +1,129 @@
+# =============================================
+#  Variables 
+# =============================================
+variable "env" {}
```

Comment:
Variables should typically be in a separate `variables.tf`. I think this is advantageous even in an example since the example gets its own directory,

Reply:
yes!